notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<h3>Create entity types</h3><p> </p><p>You can create entity types within the <code>Featurestore</code> class. The following cells create the <code>Users</code>&nbsp;entity type and&nbsp;<code>Movies</code>&nbsp;entity type.&nbsp;Process logs are printed&nbsp;in the output for each cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>